### DISCUSSION TITLE
Carregar uma listagem usando lazyload com React?

### DISCUSSION THREAD

**Discussion by renedocket** [frontendbr/forum#1728] [Perguntas e Respostas]
Olá, Pessoal! Tudo bem com vocês???

Eu gostaria de saber como que é feito um lazyload com o ReactJs, tem alguma lib, alguma maneira que vocês usam pra fazer? E como vocês fazem o controle do lazy caso essa lista tenha que ser atualizada frequentemente ?

Desde já agradeço pela ajuda! :D

**Comment by studiojms** [comment]
Segue um artigo interessante que pode ajudar: 
https://www.smashingmagazine.com/2020/03/infinite-scroll-lazy-image-loading-react/
